NitroxPatcher/Patches/Persistent/CellManager_TryLoadCacheBatchCells_Patch.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Reflection;
+using Harmony;
+using NitroxClient.MonoBehaviours;
+using UnityEngine;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.Core;
+using NitroxClient.GameLogic;
+using NitroxModel.DataStructures;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Emit;
+
+// TODO: Temporarily persistent to run before everything.  When we migrate the patch hook to an early point then make this non-persistent
+namespace NitroxPatcher.Patches.Persistent
+{
+    class CellManager_TryLoadCacheBatchCells_Patch : NitroxPatch, IPersistentPatch
```

Comment:
Let's go through each patch in this PR and add documentation at the top.  Similar to https://github.com/SubnauticaNitrox/Nitrox/blob/master/NitroxPatcher/Patches/Dynamic/Base_SpawnPiece_Patch.cs#L16

Also, we'll want to add some samples of the code that we are injecting.

Reply:
Added Some documentation